cloud/src/autogluon/cloud/predictor/multimodal_cloud_predictor.py

```diff
@@ -112,31 +107,21 @@ def predict(
         """
         test_data: str
             The test data to be inferenced.
-            Can be a pandas.DataFrame, a local path or a s3 path to a csv file.
+            Can be a pandas.DataFrame, a local path to a csv file.
             When predicting multimodality with image modality:
                 You need to specify `test_data_image_column`, and make sure the image column contains relative path to the image.
             When predicting with only images:
-                Can be a local path or a s3 path to a directory containing the images.
-                or a local path or a s3 path to a single image.
+                Can be a local path to a directory containing the images.
+                or a local path to a single image.
```

Comment:
Can be a local path to a directory containing the images or a local path to a single image.